stl/src/parallel_algorithms.cpp

```diff
@@ -157,7 +157,7 @@ static DWORD _Get_number_of_processors() noexcept {
     return _Info.dwNumberOfProcessors;
 }
 
-extern "C" {
+EXTERN_C_START

```

Comment:
These are `<Windows.h>` macros, but we conventionally use the STL's macros for this (in fact, I had to look up where this was coming from). I think it would be fine to use the STL macros here in this PR.

Reply:
this means that we should include <yvals_core.h> 
as we said above, let's file a new enhancement issue for a diff PR